vuepress/docs/v7.0/docs/README.md

```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
-
-::: tip How Google Runs Containers in Production
-Each week, Google deploys over 2 billion containers in production to run all of its services like Gmail, YouTube, and Search. Kubernetes is Google's open source solution to automate the management of containers at scale.
-:::
-
-Over 50% of Fortune 500 companies use Kubernetes with product offerings from Amazon Elastic Kubernetes Service (EKS), Google Cloud Platform (GKE), IBM (Red Hat) OpenShift, Microsoft Azure (AKS), and VMWare Pivotal (PKS).
+It is a distributed and composable environment in which components can be built, scaled and iterated independently, helping teams build agile applications that can adapt to change.
+
+![Entando Platform Development Process](./getting-started/img/concepts.png) 
+
+On the Entando Platform, development of an application is organized into four stages: Create, Curate, Compose, and Consume. In the simplest terms, you create functional building blocks, curate them into a catalog, compose the blocks onto an app or website, which is then consumed by a user.
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independent component or functional block. Developers can use the [Entando Component Generator](../create/component-gen-overview.md) and [Blueprint](../create/blueprint-features.md) to automate the building and packaging of project files, with the flexibility to customize while using software they are accustomed to. The modular design of apps built on Entando means new business capabilities can be integrated into your application without reworking existing systems. 
+
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams can share and collaborate on a single component or whole packaged business capabilities (PBCs) across organizations and applications. They can test and assess components for readiness, manage and publish versions, and share metadata for continuous upgrades. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
```

Comment:
change to Filter? none of the other verbs use "ing"

Reply:
No reason for it to match.